<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>610</width>
    <height>345</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>VentanaPrincipal</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="botongr">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>90</y>
      <width>281</width>
      <height>141</height>
     </rect>
    </property>
    <property name="text">
     <string>Iniciar sesion</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>mic.jpg</normaloff>
      <normalon>mic.jpg</normalon>mic.jpg</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>30</width>
      <height>30</height>
     </size>
    </property>
   </widget>
   <widget class="QLabel" name="label1">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>250</y>
      <width>231</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
